### PR TITLE
Easily know if a client is registered LP: #1912516

### PR DESCRIPTION
**Pull request by silverdrake11**
Link is here https://bugs.launchpad.net/landscape-client/+bug/1912516

### CODE REVIEW

File: scripts/landscape-registration-info [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply scripts/landscape-registration-info]: Yeah that is a good idea.. I am making this a part of landscape-config

File: scripts/landscape-registration-info [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+import sys, os
+if os.path.dirname(os.path.abspath(sys.argv[0])) == os.path.abspath("scripts"):
+    sys.path.insert(0, "./")
+
+from landscape.client.registration_status import display_registration_status
+
+
+if __name__ == "__main__":
+    display_registration_status()
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply scripts/landscape-registration-info]: I will add a global variable with an exit code of 5 to indicated this

File: landscape/client/tests/test_registration_status.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+class RegistrationStatusTest(LandscapeTest):
+
+    helpers = [BrokerConfigurationHelper]
+
+    def setUp(self):
+        super(RegistrationStatusTest, self).setUp()
+        self.custom_args = ['hello.py']  # Fake python script name
+        self.account_name = 'world'
+        self.data_path = '/tmp/registrationtest'
+        self.config_text = ('[client]\ncomputer_title = hello\n'
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply landscape/client/tests/test_registration_status.py]: Fixed

File: landscape/client/tests/test_registration_status.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+class RegistrationStatusTest(LandscapeTest):
+
+    helpers = [BrokerConfigurationHelper]
+
+    def setUp(self):
+        super(RegistrationStatusTest, self).setUp()
+        self.custom_args = ['hello.py']  # Fake python script name
+        self.account_name = 'world'
+        self.data_path = '/tmp/registrationtest'
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply landscape/client/tests/test_registration_status.py]: Fixed

**Review by simpoir — COMMENTED** [review]
+1 With a few more inline fixes

Please also update `man/landscape-config.txt`. You can copy the new line from `./scripts/landscape-config --help` with a new example, then test with `make manpages && man ./landscape-config.txt`

**Review by simpoir — APPROVED** [review]

**Pull request merged**